sentinel/transitions/registration.js

```diff
@@ -67,7 +67,7 @@ const booleanExpressionFails = (doc, expr) => {
     } catch (err) {
       // TODO should this count as a fail or as a real error
       logger.warn('Failed to eval boolean expression:');
-      logger.warn(err);
+      logger.warn(err.message);
```

Comment:
Did logging the `err` not work? It'd be nice to get the stacktrace. But maybe that makes it look too much like an error not a warning.

Reply:
Winston logger doesn't show stack traces. I switched to a console.log instead.

Reply:
IMO we should just drop Winston. I have no clue what it gives us that console.log doesn't.